### PR TITLE
Update Google and Facebook Open source Programs

### PR DESCRIPTION
**Pull request by tonyle9**
- Add Google Open source Program
- Add Facebook Open source Program

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Unaffiliated with the project (not self-promoting as e.g. a maintainer, creator, contractor or employee)

************EDITING AN EXISTING TOPIC OR COLLECTION************

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC************

- [ ] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:

************CURATING A NEW COLLECTION************

- [ ] I've formatted my changes as a new folder directory, named for the collection as it appears in the URL on GitHub (e.g. `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this collection should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by talsafran** [comment]
Hi @tonyle9 – thanks for keeping our content fresh! I reopened the PR since it was pointing to the `dependabot/bundler/rubocop-0.56.0` branch instead of `master`. LGTM!

**Comment by talsafran** [comment]
@tonyle9 – looks like [ci is failing](https://travis-ci.org/github/explore/builds/390987049?utm_source=github_status&utm_medium=notification) because the Facebook repo isn't publically accessibile:

```

  1) Failure:
collections::open-source-organizations collection#test_0005_fails if a repository does not exist or is private [/home/travis/build/github/explore/test/collections_test.rb:56]:
Expected ["open-source-organizations: Facebook/facebook.github.io does not exist or is private"] to be empty.
5334 runs, 111409 assertions, 1 failures, 0 errors, 0 skips
rake aborted!
Command failed with status (1)
/home/travis/build/github/explore/vendor/bundle/ruby/2.4.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.4.1/bin/bundle:23:in `load'
/home/travis/.rvm/gems/ruby-2.4.1/bin/bundle:23:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)
Inspecting 8 files
........
````

Could you fix that issue please and merge in latest master? Thanks 🙌

File: collections/open-source-organizations/index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - Google/google.github.io
```
Comment (brandonrosage) [inline collections/open-source-organizations/index.md]: I'm not seeing the value of including [this repo](https://github.com/google/google.github.io). It just redirects to the Google organization. Help me out here.

File: collections/open-source-organizations/index.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ items:
...
+ - Facebook/facebook.github.io
```
Comment (brandonrosage) [inline collections/open-source-organizations/index.md]: This repo doesn't appear to exist: https://github.com/Facebook/facebook.github.io

File: collections/open-source-organizations/index.md (hunk 1 above)
Comment (faizan900) [reply collections/open-source-organizations/index.md]: https://youtu.be/JrF2npRKIbQ

**Comment by talsafran** [comment]
Closing this branch for inactivity.

**Pull request closed without merging**